server.js

```
@@ -1,66 +0,0 @@
-const express = require('express');
-const server = express();
-const cors = require('cors');
-const admin = require('./firebase-admin/admin');
-const db = require('./db/dbConfig');
-const todosRouter = require('./todos-router/todos-router.js');
-require('dotenv').load();
-
-server.use(express.json());
-server.use(cors());
-
-// Middleware function to verify Firebase token
-// Adds the user's UID to the request body
-async function verifyToken(req, res, next) {
-  const idToken = req.headers.authorization;
-
-  try {
-    const decodedToken = await admin.auth().verifyIdToken(idToken);
-
-    if (decodedToken) {
-      req.body.uid = decodedToken.uid;
-
-      return next();
-    } else {
-      return res.status(401).send('You are not authorized!');
-    }
-  } catch (e) {
-    return res.status(401).send('You are not authorized!');
-  }
-}
-
-server.use('/', verifyToken);
-server.use('/api', todosRouter);
-
-server.get('/', (req, res) => {
-  res.status(200).send('Hello Yaro!');
-});
-
-// Check if the user exists in the DB. If not, insert the user
-server.post('/verifyregistration', async (req, res) => {
-  const { uid } = req.body;
-  const user = {
-    uid,
-  };
-
-  try {
-    const data = await db('users').where('uid', uid);
-
-    if (data.length > 0) {
-      res.status(200).json(data);
-    } else {
-      const id = await db('users').insert(user);
-
-      if (id) {
```

If it possible that `id` doesn't exist? I would expect that if the `insert` fails, an error will be thrown, rather than the function returning `null`.

In general, avoid writing "just in case" code. Check the documentation (or write some tests) to verify what the code will actually do when something goes wrong, and then plan for that accordingly. It's possible that `id` will be null and this `if` block is necessary, but you should verify that first before planning for it.